README.md

```diff
@@ -36,7 +36,9 @@ $ cd ~/rosbag_ws/src
 Clone this repository into the source folder:
 
 ```
-$ git clone https://github.com/ros2/rosbag2.git
+$ git clone https://github.com/ros2/rosbag2.git # if you are building all of ROS2 from source.
+
+$ git clone --branch $ROS_DISTRO https://github.com/ros2/rosbag2.git # if you are just building rosbag2 from source.
```

Comment:
While we can generally assume that there is a branch existing according to the distribution, we can't guarantee this. I would propose to highlight that in an overlay compilation (that is if you're only building rosbag2 on top of a debian installation) one has to checkout the right branch according to the base installation.

Reply:
Hi @Karsten1987,
Thank you for quick clarification.

I have changed the documentation update according to your feedback. Please check to see if update is appropriate.